Python3.py.jinja

```diff
@@ -0,0 +1,21 @@
+{{ id }}_model = genn_model.create_custom_neuron_class(
```

Comment:
when working with jinja2 in the past I found it useful to add a comment at the top which python variables are expected to be available for use for the template to work, i.e. something like
```
{# Uses variables: parameters, variables, ... #}
```

Reply:
Yes, good point. Fixed in https://github.com/GenGeNN/GeNN-neuron-lang-bindings/pull/1/commits/7919e3820f2ebc48f536ca4c33cb255b96289312